policyengine_us/parameters/gov/states/al/tax/income/income_sources.yaml

```diff
@@ -0,0 +1,24 @@
+description: Alabama counts these income sources for the state adjusted gross income.
+values:
+  2021-01-01:
+    - employment_income
+    - self_employment_income
+    - interest_income
+    - dividend_income
+    - alimony_income
+    - regular_ira_distributions
+    - roth_ira_distributions
+    - pension_income
+    - rental_income
+
+metadata:
+  unit: list
+  period: year
+  label: Alabama adjusted gross income income sources
+  reference:
+    - title: 2022 Alabama Department of Revenue 
+      href: https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2023/01/22f40bk.pdf#page=8
```

Comment:
i dont see these income sources listed here

Reply:
Added the actual form 40